src/components/molecules/Human/Human.tsx

```diff
@@ -0,0 +1,11 @@
+import Categories from "@app/components/molecules/Categories/Categories";
+import { categories } from "@app/components/molecules/Human/config";
+
+const Human = () => (
```

Comment:
Are you sure you want to create a separate file for each category? That's gonna be a lot of files. Maybe it would be more useful to create a generic component for all categories, and pass that stuff via props. By doing that, you could separate UI from the content.